ansible_collections/arista/avd/roles/eos_designs/doc/custom-structured-configuration.md

```diff
@@ -147,7 +146,7 @@ my_dci_ethernet_interfaces:
     peer_interface: Ethernet123
     peer_type: my_precious
 
-my_special_dci_ethernet_interfaces:
+
```

Comment:
remove extra empty line

Reply:
actually this line (and the one above) should not have been removed.. fixing this